AR-BoardGame/AR-BoardGame/AR_BoardGameApp.swift

```diff
@@ -10,17 +10,22 @@ import SwiftUI
 @main
 struct AR_BoardGameApp: App {
     
-    @State private var contentViewMdoel = ContentViewModel()
+    @State private var contentViewModoel = ContentViewModel()
+    @State private var timerViewModel = TimerViewModel()
     
     var body: some Scene {
-        WindowGroup {
-            ContentView()
-                .environment(contentViewMdoel)
+        WindowGroup(id: "ContentWindow") {
+            ContentView(contentViewModel: contentViewModoel)
         }.windowStyle(.volumetric)
 
         ImmersiveSpace(id: "ImmersiveSpace") {
             ImmersiveView()
```

Comment:
id들을 한곳에서 관리해주면 좀 더 직관적이고 편할 거 같습니다
```suggestion

// SceneID.swift
enum SceneID {
  enum WindowGroup {
    case content
    case timer

    var id: String {
      switch self {
      case .content: return "Content"
      case .timer: return "Timer"
      }
    }
  }
  
  enum ImmersiveSpace {
    case game
    
    var id: String {
      switch self {
      case .game: return "Game"
      }
    }
  }
}

// AR_BoardGameApp.swift
//  ...
        WindowGroup(id: SceneID.WindowGroup.content.id) {
            ContentView(contentViewModel: contentViewModoel)
        }.windowStyle(.volumetric)

        ImmersiveSpace(id: SceneID.ImmersiveSpace.game.id) {
            ImmersiveView()
```